packages/common/src/authorization/organizationMemberAbility.test.ts

```diff
@@ -833,8 +833,8 @@ describe('Organization member permissions', () => {
                 expect(ability.can('manage', 'SqlRunner')).toEqual(false);
             });
 
-            it('can use the SemanticViewer', () => {
-                expect(ability.can('manage', 'SemanticViewer')).toEqual(true);
+            it('can view the SemanticViewer', () => {
```

Comment:
we never used the 'manage' permission anywhere in the app - so simplified this to view